### PR TITLE
code template syntax fix

### LINKED ISSUE #972: Broken code in getting started doc page
## Description
Noticed a broken code on the getting started doc page. This should be as a result of the omission of a closing tag.

## Screenshots
image

- [ ] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by Akpjunior94**
## Fixes

Fixes #972 by @Akpjunior94 

## Description

Some HTML tag syntax templates are not well-formatted on the page. This pull request fixes that.

## Screenshots

Problems
image
image

Solutions
image
image

## Checklist
 
- [x] Added spaces where necessary.
- [x] I confirmed the project locally and confirmed there are no visible errors
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by MuluhGodson** [comment]
Hi @Akpjunior94 
I think my assumptions and hypothesis in #1000  may be faulty. 
If what I said was correct then this PR should have triggered all the actions given that you have opened a PR on `main` and you have made changes in the `src` folder.

**Comment by Akpjunior94** [comment]
> Hi @Akpjunior94 I think my assumptions and hypothesis in #1000 may be faulty. If what I said was correct then this PR should have triggered all the actions given that you have opened a PR on `main` and you have made changes in the `src` folder.

Yes @MuluhGodson yeah It should because I made changes to the `src` and sent a PR to the `main` branch

**Comment by MuluhGodson** [comment]
I think the best thing to do is update your own fork with the latest changes from the main repo. That way changes are pulled and your fork's main is current with the main repo. 
 
```bash
git checkout <commits_branch>
git merge origin/main
#solve merge conflicts 
git commit 
git push 
#pull request git push
```

<commits_branch> is the branch you did your updates in.

**Comment by Akpjunior94** [comment]
> I think the best thing to do is update your own fork with the latest changes from the main repo. That way changes are pulled and your fork's main is current with the main repo.
> 
> ```shell
> git checkout <commits_branch>
> git merge origin/main
> #solve merge conflicts 
> git commit 
> git push 
> #pull request git push
> ```
> 
> <commits_branch> is the branch you did your updates in.

Does that mean I have to close this PR do follow the above step on my vscode terminal and push back to my forked then create the PR again?

**Comment by brylie** [comment]
> Does that mean I have to close this PR do follow the above step on my vscode terminal and push back to my forked then create the PR again?

@Akpjunior94, you should be able to leave your pull request open and just run the commands in your terminal. The pull request will be updated when you run `git push`.

**Comment by brylie** [comment]
Sorry. I accidentally clicked "close with comment" instead of "comment".

**Review by brylie — CHANGES_REQUESTED** [review]
Please upgrade your npm to version 8, so we don't have so many changes to package-lock.json.

**Comment by brylie** [comment]
Also, what version of `npm` do you have installed @Akpjunior94? It uses an outdated format for `package-lock.json`, which creates a lot of line changes in this pull request. PleaWould you please updater npm to version 8.?

**Comment by Akpjunior94** [comment]
> Please upgrade your npm to version 8, so we don't have so many changes to package-lock.json.

I have done that. I actually did after sending the PR yesterday. I forgot to do it before sending it. I wanted to be sure how I can go abt updating the PR.

**Comment by Akpjunior94** [comment]
> Also, what version of `npm` do you have installed @Akpjunior94? It uses an outdated format for `package-lock.json`, which creates a lot of line changes in this pull request. PleaWould you please updater npm to version 8.?

version 8.00

**Comment by brylie** [comment]
OK, sounds good. Re-run the command `npm install` with the new version of `npm` and commit the changes to package-lock.json, so fewer lines are changed (if any).

**Comment by Akpjunior94** [comment]
> OK, sounds good. Re-run the command `npm install` with the new version of `npm` and commit the changes to package-lock.json, so fewer lines are changed (if any).

Okay

**Comment by Akpjunior94** [comment]
It's updated @brylie you can check the PR.

File: package-lock.json [hunk 1]
```diff
@@ -64663,4 +64663,4 @@
       "dev": true
     }
   }
-}
+}
```
Comment (brylie) [inline package-lock.json]: Add newline at end
```suggestion
}

```

**Review by brylie — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (Akpjunior94) [reply package-lock.json]: Okay

File: src/components/cards/cards.stories.mdx [hunk 2]
```diff
@@ -45,7 +46,7 @@ export const cardWithButton = (width, image_size, orientation) => `<div style="m
   `}</Story>
 </Canvas>
 
-export const postVideo = (width,image_size,orientation) => `<div style="max-width: ${width}px;">
+export const postVideo = (width,image_size,orientation) => `<div style=" max-width: ${width}px;">
```
Comment (brylie) [inline src/components/cards/cards.stories.mdx]: Lint
```suggestion
export const postVideo = (width,image_size,orientation) => `<div style="max-width: ${width}px;">
```

File: src/components/cards/cards.stories.mdx [hunk 3]
```diff
@@ -85,7 +86,7 @@ export const postVideo = (width,image_size,orientation) => `<div style="max-widt
   <Story name="PostVideoLarge" parameters={{
     design: figmaConfig('776%3A86')
   }}>{`
-    <div style="width: 100%;">
+    <div style=" max-width: 100%;">
```
Comment (brylie) [inline src/components/cards/cards.stories.mdx]: Lint
```suggestion
    <div style="max-width: 100%;">
```

File: src/components/cards/cards.stories.mdx [hunk 4]
```diff
@@ -107,7 +108,7 @@ export const postVideo = (width,image_size,orientation) => `<div style="max-widt
 
 
 export const postEvent = (width) => `<div
-style="max-width: ${width}px;">
+style=" max-width: ${width}px;">
```
Comment (brylie) [inline src/components/cards/cards.stories.mdx]: Lint
```suggestion
style="max-width: ${width}px;">
```

File: src/components/cards/cards.stories.mdx [hunk 5]
```diff
@@ -148,7 +149,7 @@ style="max-width: ${width}px;">
   <Story name="PostStatistic" parameters={{
     design: figmaConfig('776%3A144')
   }}>{`
-    <div style="max-width: 600px;">
+    <div style=" max-width: 600px;">
```
Comment (brylie) [inline src/components/cards/cards.stories.mdx]: Lint
```suggestion
    <div style="max-width: 600px;">
```

File: src/components/cards/cards.stories.mdx [hunk 6]
```diff
@@ -171,7 +172,7 @@ style="max-width: ${width}px;">
   <Story name="PostQuote" parameters={{
     design: figmaConfig('776%3A3')
   }}>{`
-    <div style="max-width: 600px;">
+    <div style=" max-width: 600px;">
```
Comment (brylie) [inline src/components/cards/cards.stories.mdx]: Lint
```suggestion
    <div style="max-width: 600px;">
```

File: src/components/cards/cards.stories.mdx [hunk 7]
```diff
@@ -197,7 +198,7 @@ style="max-width: ${width}px;">
   <Story name="ImageTop" parameters={{
     design: figmaConfig('776%3A3')
   }}> {`
-    <div style="max-width: 340px;">
+    <div style=" max-width: 340px;">
```
Comment (brylie) [inline src/components/cards/cards.stories.mdx]: Lint
```suggestion
    <div style="max-width: 340px;">
```

File: src/components/cards/cards.stories.mdx [hunk 8]
```diff
@@ -260,7 +261,7 @@ export const linkBorderCard = `<article class="card entry-post link">
   <Story name="Blog entry horizontal" parameters={{
     design: figmaConfig('776%3A3')
   }}>{`
-    <div style="max-width: 384px;" class="margin-normal">
+    <div style=" max-width: 384px;" class="margin-normal">
```
Comment (brylie) [inline src/components/cards/cards.stories.mdx]: Lint
```suggestion
    <div style="max-width: 384px;" class="margin-normal">
```

File: src/stories/meta/gettingstarted.stories.mdx [hunk 9]
```diff
@@ -102,7 +102,9 @@ Let's start by adding in a `Global Header` & a `Header` component. Note that the
         </div>
       </nav>
     </header>
-    <div style="background-color: rgb(237, 89, 47); height: 200px;"></div>
+    <div style=" background-color:red; height: 200px;"></div>
```
Comment (brylie) [inline src/stories/meta/gettingstarted.stories.mdx]: Lint
```suggestion
    <div style="background-color:red; height: 200px;"></div>
```

File: src/stories/meta/gettingstarted.stories.mdx [hunk 10]
```diff
@@ -102,7 +102,9 @@ Let's start by adding in a `Global Header` & a `Header` component. Note that the
...
+
+    style="background-color: rgb(237, 89, 47); height: 200px;"
```
Comment (brylie) [inline src/stories/meta/gettingstarted.stories.mdx]: Remove floating code
```suggestion
```

**Review by brylie — CHANGES_REQUESTED** [review]

File: src/components/cards/cards.stories.mdx [hunk 11]
```diff
@@ -9,7 +9,8 @@ export const content = `<div class="content">
             Description metus ligula pharetra litora class massa viverra, facilisis cum aenean hendrerit eget magnis convallis.
           </div>`
 
-export const cardWithButton = (width, image_size, orientation) => `<div style="max-width: ${width}px;">
+export const cardWithButton = (width, image_size, orientation) => 
+`<div style=" max-width: ${width}px;">
```
Comment (brylie) [inline src/components/cards/cards.stories.mdx]: Lint
```suggestion
`<div style="max-width: ${width}px;">
```

**Review by brylie — CHANGES_REQUESTED** [review]

File: src/stories/meta/gettingstarted.stories.mdx [hunk 12]
```diff
@@ -102,7 +102,7 @@ Let's start by adding in a `Global Header` & a `Header` component. Note that the
         </div>
       </nav>
     </header>
-    <div style="background-color: rgb(237, 89, 47); height: 200px;"></div>
+    <div style="background-color:red; height: 200px;"></div>
```
Comment (brylie) [inline src/stories/meta/gettingstarted.stories.mdx]: Revert this change
```suggestion
    <div style="background-color: rgb(237, 89, 47); height: 200px;"></div>
```

**Review by brylie — CHANGES_REQUESTED** [review]

**Comment by brylie** [comment]
@Akpjunior94, please be sure to use meaningful commit messages that describe the change you are making. Where possible, add related changes under a single commit. For example, you can make all of the lint changes with a single commit.

**Comment by Akpjunior94** [comment]
> @Akpjunior94, please be sure to use meaningful commit messages that describe the change you are making. Where possible, add related changes under a single commit. For example, you can make all of the lint changes with a single commit.

Okay Noted

**Review by brylie — APPROVED** [review]

**Comment by brylie** [comment]
@all-contributors please add @Akpjunior94 for code

**Pull request merged**